packages/terra-consumer-nav/src/components/smart-link/SmartLink.jsx

```diff
@@ -25,19 +25,30 @@ const SmartLink = ({
   handleClick,
   children,
   ...customProps
-}) => (
-  (isExternal || target === '_blank') ?
-    <a {...customProps} onClick={handleClick} target={target} href={url}>{children}</a> :
+}) => {
+  if (isExternal || target === '_blank') {
+    return (
+      <a {...customProps} onClick={handleClick} target={target} href={url}>{children}</a>
+    );
+  }
+  const a = document.createElement('a');
+  a.href = url;
```

Comment:
can we set href while creating  element ?

`document.createElement('a', {href: url}); `

Reply:
that is a react create element api. It doesn't seem to work for document create element
```
const x = document.createElement('a', {href: 'test'});

undefined
x
<a>​</a>​
```